Automattic/bash-cache-buildkite-plugin
# Add a script to install node packages using npm

## What it does

Add a script to install Node JS packages using `npm`.

## How to test

- Ensure that you have a working installation of Node JS
- Execute the script (from the same location as `package.json`)  to install the node packages.

## Who

@Automattic/apps-infrastructure
